File: Estrutura_Repetição/C#/Ex1101/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        int x, y;

        while (true) {
            string[] inputs = Console.ReadLine().Split();
            x = Convert.ToInt32(inputs[0]);
            y = Convert.ToInt32(inputs[1]);

            int menor = Math.Min(x, y);
            int maior = Math.Max(x, y);

            if (menor <= 0 || maior <= 0) {
                break;
            } else {
                int soma = 0;
                for (int i = menor; i < maior + 1; i++) {
                    soma += i;
                    Console.Write(i + " ");
                }
                Console.WriteLine("Sum=" + soma);
            }
        }
    }
}

File: Estrutura_Repetição/Java/Ex1101/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int x, y;

        while (true) {
            x = sc.nextInt();
            y = sc.nextInt();

            int menor = Math.min(x, y);
            int maior = Math.max(x, y);

            if (menor <= 0 || maior <= 0) {
                break;
            } else {
                int soma = 0;
                for (int i = menor; i < maior + 1; i++) {
                    soma += i;
                    System.out.print(i + " ");
                }
                System.out.println("Sum=" + soma);
            }
        }

        sc.close();
    }
}